src/tactic/ui/widget/embed_wdg.py

```diff
@@ -66,7 +66,9 @@ def get_display(my):
         #div = DivWdg()
```

Comment:
should define ARGS_KEYS =  {
'video_class': .....  refer to table_layout_wdg.py
}